llama/model.py

```diff
@@ -205,10 +205,20 @@ def forward(
 
         # # Activation output sharding
         # # TODO(yeounoh) remove this after activation sharding support is enabled.
-        num_devices = xr.global_runtime_device_count()
-        device_ids = torch.arange(num_devices)
-        data_model_mesh = xs.Mesh(device_ids, (4, 1, 2))
-        xs.mark_sharding(output, data_model_mesh, (0, 1, 2), use_dynamo_custom_op=True)
+        # num_devices = xr.global_runtime_device_count()
```

Comment:
@wonjoolee95  can you add a discussion on the reason to keep `output = self.wo(output)` when we want to do the `dynamo_mark_sharding` call  after?

Reply:
This is the original implementation, and we are just adding the annotations.